Tests/BondTests/UICollectionViewTests.swift

```diff
@@ -99,10 +99,48 @@ class UICollectionViewTests: XCTestCase {
             array.insert(1, at: 0)
         }
 
-        let possibleResultA = [OrderedCollectionDiff(), OrderedCollectionDiff<IndexPath>(inserts: [IndexPath(row: 1, section: 0), IndexPath(row: 0, section: 0)])]
-        let possibleResultB = [OrderedCollectionDiff(), OrderedCollectionDiff<IndexPath>(inserts: [IndexPath(row: 0, section: 0), IndexPath(row: 1, section: 0)])]
+        let possibleResultA = [OrderedCollectionDiff(reload: true), OrderedCollectionDiff<IndexPath>(inserts: [IndexPath(row: 1, section: 0), IndexPath(row: 0, section: 0)])]
+        let possibleResultB = [OrderedCollectionDiff(reload: true), OrderedCollectionDiff<IndexPath>(inserts: [IndexPath(row: 0, section: 0), IndexPath(row: 1, section: 0)])]
         XCTAssert(collectionView.observedEvents == possibleResultA || collectionView.observedEvents == possibleResultB)
     }
+
+    func testBatchUpdatesEmptyBatch() {
+        array.batchUpdate { (array) in }
+        let expectedResult = [OrderedCollectionDiff<IndexPath>(reload: true)] // It Should reload data only once
```

Comment:
```suggestion
        let expectedResult = [OrderedCollectionDiff<IndexPath>(reload: true)] // It should reload data only once
```

Can you explain "It" - this comment isn't very clear. Perhaps the comment should explain _why_ the current change needs to reload data one time?